packages/ng/demo/app/selects/expert-select/expert-select.component.html

```diff
@@ -0,0 +1,18 @@
+<h1>With Options</h1>
+<div class="textfield mod-select mod-material" [class.mod-remove]="_canRemove">
+	<div [luSelect]="selectRefOptions"
+		[(ngModel)]="itemOptions"
+		class="textfield-input"
+		name="itemOptions"
+		luEmpty
+		clearable="true"
+		(canremove)="canRemove($event)"></div>
+	<label class="textfield-label" for="itemOptions">my label</label>
+	<lu-select-popover #selectRefOptions>
+		<lu-select-option *ngFor="let option of options" [value]="option">{{option.name}}</lu-select-option>
+	</lu-select-popover>
+	<button class="actionIcon" (click)="clear()">
```

Comment:
Your right ! At first, I found intresting to open this behaviour to developers but after your comment, i can indeed easly refactor the "clearable" behaviour to hide it in the luSelectDirective. It will simplify the mecanism 👍

Reply:
about textfield-actionIcon, @sanlucca and @jeremie-lucca what did you think ?

Reply:
@lucienbertin You're right, having only an "actionIcon" (which is a very generic scss component) is not satisfying.
I'm still thinking about the best way to deal with multiple scss components (and it's naming convention) in an ng component, especially for icons which can have multiple occurences. 
Maybe we could tolerate a double "textfield-empty actionIcon" class in those cases. 🤔